### PR TITLE
feat(approvals-satisfied): comment when PR does not meet approvals

### PR DESCRIPTION
**Pull request by sjschmidt93**
### :pencil: Description
- 

### :link: Related Issues
-

### CODE REVIEW

File: src/helpers/approvals-satisfied.ts
Comment (danadajian) [inline src/helpers/approvals-satisfied.ts:102]: This helper doesn't only run in the context of the merge queue, so we should make the message more generic. Maybe like 
```
Required approvals not satisfied: <logsJoined>
```
and we could pass the existing `body` input into this helper so the merge queue invocation can provide a merge queue-specific message too

File: src/helpers/approvals-satisfied.ts
Comment (sjschmidt93) [inline src/helpers/approvals-satisfied.ts:97]: never heard of unshift until now, but it is equal to `['Required approvals not satisifed', ...logs]`

File: src/helpers/approvals-satisfied.ts
Comment (danadajian) [inline src/helpers/approvals-satisfied.ts:36]: Can we use the existing `body` input instead of this second argument for providing an optional comment body? `approvalsSatisfied` is a standalone action

File: src/helpers/approvals-satisfied.ts
Comment (danadajian) [inline src/helpers/approvals-satisfied.ts]: Can we rename to `body` and add this to the above interface?

**Review by danadajian — APPROVED** [review]

**Pull request merged**
